source/common/compressor/zlib_compressor_impl.h

```diff
@@ -50,7 +50,7 @@ class ZlibCompressorImpl : public Zlib::Base, public Compressor {
     Filtered = 1,
     Huffman = 2,
     Rle = 3,
-    Standard = 4,
+    Standard = 0,
```

Comment:
Please add a comment on why this is done this way. Also, does this need a release note?

Reply:
I put the defines from zlib.h instead of magic numbers. I think they are self-descriptive enough.